src/material-experimental/mdc-button/_button-base.scss

```diff
@@ -2,7 +2,7 @@
 @use '@angular/material' as mat;
 @use '@material/touch-target' as mdc-touch-target;
 
-@use '../mdc-helpers/mdc-helpers';
+@use '../mdc-core/mdc-helpers/mdc-helpers';
```

Comment:
Maybe we should have an `_index.scss` inside `mdc-helpers` in order to avoid imports like this?

Reply:
We should probably just get rid of MDC helpers. It looks like all stuff related to typography and theming so it can be moved to mdc-core/typography or mdc-core/theming. In this PR though I'm just going to leave it